package.json

```diff
@@ -60,6 +63,8 @@
   },
   "devDependencies": {
     "@ledgerhq/logs": "^6.10.1",
+    "@types/chroma-js": "^2.4.0",
+    "@types/ellipsize": "^0.1.1",
```

Comment:
```suggestion
```

Reply:
applied change, pending debug of ellipsizeFN alternative

Reply:
Can this be removed now?